pkg/client/getstdin/getstdin.go

```diff
@@ -0,0 +1,17 @@
+package getstdin
```

Comment:
sorry to be an annoying guy but, you don't need to create a new package, just `client.GetStdin()` looks good to me. (if you move this file to `pkg/client/getstdin.go` and change the package name to `client` will be great)

Reply:
no problem man.